src/Cms/LanguageRouter.php

```diff
@@ -106,26 +91,32 @@ public function routes(): array
 	 * Wrapper around the Router::call method
 	 * that injects the Language instance and
 	 * if needed also the Page as arguments.
-	 *
-	 * @param string|null $path
-	 * @return mixed
 	 */
-	public function call(string $path = null)
+	public function call(string $path = null): mixed
 	{
-		$language = $this->language;
-		$kirby    = $language->kirby();
-		$router   = new Router($this->routes());
+		$language       = $this->language;
+		$kirby          = $language->kirby();
+		$this->router ??= new Router($this->routes());
```

Comment:
Caching router instance is intended?

Reply:
We always had the property in the class but it was never used, thought this probably was supposed to always be cached